WalletWasabi/CoinJoin/Client/Clients/CoinJoinClient.cs

```diff
@@ -27,6 +27,7 @@
 using static WalletWasabi.Crypto.SchnorrBlinding;
 
 namespace WalletWasabi.CoinJoin.Client.Clients;
+#pragma warning disable
```

Comment:
top of the file?

Reply:
IMO it looks lame on the top. Until here there are just usings and the namespace, no possible warnings there.
From here, as the class opens, the pragma should as well, indicating that all the warnings in the class under this pragma should be disabled. 
But that just my honest opinion, idk if it makes sense too you.

Reply:
Well, it's fine by me. Others might have a different opinion.